src/EntityFramework6.Npgsql/NpgsqlMigrationSqlGenerator.cs

```diff
@@ -501,6 +499,21 @@ protected virtual void Convert(DropPrimaryKeyOperation dropPrimaryKeyOperation)
 
         #region Misc functions
 
+        /// <summary>
+        /// Quotes an identifier for Postgres SQL
+        /// </summary>
+        /// <param name="identifier"> The identifier to be quoted. </param>
+        /// <returns> The quoted identifier. </returns>
+        protected virtual string QuoteIdentifier(string identifier)
```

Comment:
Why is this virtual ? It will prevent the compiler from inlining it and reduce performance (it is a hot path).